tox.ini

```diff
@@ -1,9 +1,16 @@
 [tox]
 envlist =
-    py{27,34,35}-sphinx{13,14,15}
+    py{27,34,35,36}-sphinx{13,14,15}
```

Comment:
It's a super old version of Sphinx. I'd rather ask what the value is of supporting 1.3?